app/javascript/packs/baseInitializers.js

```diff
@@ -8,7 +8,10 @@ import {
 
 initializeCommentDate();
 initializeCommentPreview();
-initializeNotifications();
+
```

Comment:
```suggestion
initializeNotifications();

```

I believe this also needs to be called on the initial load _in addition_ to InstantClick